sysid-application/src/main/native/include/sysid/view/Analyzer.h

```diff
@@ -40,7 +41,23 @@ namespace sysid {
 class Analyzer : public glass::View {
  public:
   /**
-   * The different motor controlle rtiming presets that can be used.
+   * The different display and processing states for the GUI
+   */
+  enum GUIState {
```

Comment:
Use an `enum class` instead of an `enum` and wpilib's C++ enum value naming style. For example, `kWaitingForJSON` instead of `WAITING_FOR_JSON`.